Makefile

```diff
@@ -75,3 +122,38 @@ eclint/lint: | guard/program/eclint
 	@ echo "[$@]: Linting all files with eclint..."
 	eclint check
 	@ echo "[$@]: All files PASSED lint test!"
+
+docs/%: FOUND_DIR ?= find ${@F} -type d \( -name "lx*" -o -name "win*" \)
+docs/%: README_DIR ?= .
+docs/%: README_BASE ?= _docs/BASE.md.jinja2
+docs/%: README_MAIN := $(README_DIR)/MAIN.md
+docs/%: README_PARTS ?= $(README_MAIN) <(echo) <(./scripts/terraform-docs.sh markdown table $(README_DIR))
+docs/%: README_FILE ?= $(README_DIR)/README.md
+
+docs/template:
+	jinja2 $(README_BASE) -D name=$(README_DIR) > $(README_MAIN)
+
+docs/lint: | guard/program/terraform-docs
+	@ echo "[$@]: Linting documentation files.."
```

Comment:
It would help in the output if these two `echo` lines referenced `README_DIR` somehow to make it more clear _which_ docs were linted... could put it in the leading marking, or wherever...

```
	@ echo "[$@/$(README_DIR)]: ...
```

Same goes for `docs/generate`